test/unit/test_exports/test_derived_quantities/test_point_value.py

```diff
@@ -0,0 +1,34 @@
+from festim import PointValue
+import fenics as f
+import pytest
+
+
+def test_title_H():
+    x = 1
+    field = "solute"
+    my_value = PointValue(field, x)
+    assert my_value.title == "{} value at [{}]".format(field, x)
+
+
+def test_title_T():
+    x = 1
+    field = "T"
+    my_value = PointValue(field, x)
+    assert my_value.title == "{} value at [{}]".format(field, x)
+
+
+class TestCompute:
+    mesh = f.UnitIntervalMesh(10)
+    V = f.FunctionSpace(mesh, "P", 1)
+
+    c = f.interpolate(f.Expression("x[0]", degree=1), V)
+
+    x = 1
+    my_value = PointValue("solute", x)
+    my_value.function = c
+
+    def test_minimum(self):
+        expected = self.c(self.x)
+
+        produced = self.my_value.compute()
+        assert produced == expected
```

Comment:
Could we also test try to test this on a multi-dimensional mesh?

Reply:
@RemDelaporteMathurin you mean 2D/3D?

Reply:
Yes

Reply:
Can it me done as:
```python
@pytest.mark.parametrize(
    "mesh,x", [(f.UnitIntervalMesh(10), 1), (f.UnitCubeMesh(10, 10, 10), (1, 0, 1))]
)
def test_point_compute(mesh, x):
    V = f.FunctionSpace(mesh, "P", 1)
    c = f.interpolate(f.Expression("x[0]", degree=1), V)

    my_value = PointValue("solute", x)
    my_value.function = c

    expected = c(x)
    produced = my_value.compute()
    assert produced == expected
``` 
or the `TestCompute` class should remain?